src/Commands/DatabaseSanitizeCommands.php

```diff
@@ -2,21 +2,30 @@
 
 namespace Drupal\database_sanitize\Commands;
 
+use Drupal\database_sanitize\DatabaseSanitize;
 use Drush\Commands\DrushCommands;
 
 /**
- * A Drush commandfile.
- *
- * In addition to this file, you need a drush.services.yml
- * in root of your module, and a composer.json file that provides the name
- * of the services file to use.
- *
- * See these files for an example of injecting Drupal services:
- *   - http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php
- *   - http://cgit.drupalcode.org/devel/tree/drush.services.yml
+ * Class DatabaseSanitizeCommands
```

Comment:
I'm sure we can do better than this :smiley: 
```suggestion
 * Drush commands for Database Sanitize.
```